server/src/main/java/io/crate/auth/JWTAuthenticationMethod.java

```diff
@@ -51,18 +57,26 @@
 public class JWTAuthenticationMethod implements AuthenticationMethod {
 
     public static final String NAME = "jwt";
+    public static final String ISS_KEY = "iss";
+    public static final String AUD_KEY = "aud";
 
     private final Roles roles;
 
+    private final Map<String, String> jwtDefaults;
+
     private final Function<String, JwkProvider> urlToJwkProvider;
 
     private final Supplier<String> clusterId;
 
 
     public JWTAuthenticationMethod(Roles roles,
+                                   Settings settings,
                                    Function<String, JwkProvider> urlToJwkProvider,
                                    Supplier<String> clusterId) {
         this.roles = roles;
+        this.jwtDefaults = new HashMap<>();
+        jwtDefaults.put(ISS_KEY, settings.get(AUTH_HOST_BASED_JWT_ISS_SETTING.getKey()));
+        jwtDefaults.put(AUD_KEY, settings.get(AUTH_HOST_BASED_JWT_AUD_SETTING.getKey()));
```

Comment:
Why the indirection via the map? You can either use a field per setting value or keep a reference to the settings and retrieve it on access below.

I think there's even a settings.get variant that takes a fallback.